README.md

```diff
@@ -1,6 +1,10 @@
 srpms
 =====
 
-Source rpms to feed to copr: copr.fedoraproject.org
+Source rpms to feed to copr: [copr.fedoraproject.org](http://copr.fedoraproject.org)
```

Comment:
Small change, `Copr` is the way it's written so we should capitalize it unless it's a URL, directory path, etc.